open-craft/frontend-app-authn
# chore: port code drift to quince

### Description

Cherry-pick of the following commits which did not make it to upstream `Quince` release:

| Commit Id | Commit Title | Upstream PR | Ticket |
|-|-|-|-|
| [78722f3](https://github.com/openedx/frontend-app-authn/commit/78722f3e7332ef321cc49f8ec34842379c429362) | feat: implement SHOW_REGISTRATION_LINKS setting | #1066 | [BB-7843](https://tasks.opencraft.com/browse/BB-7843) |
| [abad25c](https://github.com/open-craft/frontend-app-authn/commit/abad25c00348481b4f2a9350c62828f845ab9189) | feat: redirect to custom URL when third-party auth account is unlinked | #1078 | [BB-7742](https://tasks.opencraft.com/browse/BB-7742)


OpenCraft Internal Ticket : [BB-8369](https://tasks.opencraft.com/browse/BB-8369)

### Testing

This branch is deployed to [this sandbox](https://kaustav-sandbox.staging.do.opencraft.hosting)
1. Make sure "Registration" tab is missing from the authn MFE
2. Login using cognito using the credentials `testuser : testuser`. The authn MFE should redirect to a third party url due to unlinked cognito account
